phoenix-core-client/src/main/java/org/apache/phoenix/util/ViewUtil.java

```diff
@@ -886,4 +893,44 @@ public static void addTagsToPutsForViewAlteredProperties(List<Mutation> tableMet
 
         }
     }
+
+    /**
+     * Retrieves the list of unique view index ids from SYSTEM.CATALOG for a given view index table
+     * @param connection - Phoenix Connection
+     * @param tableName - name of the view index table for which view index ids need to be retrieved
```

Comment:
It looks like this is the physical name of the view Index table and not the logical name. Can you update the API document to clarify this.